test/e2e/kubectl/kubectl.go

```diff
@@ -1290,23 +1291,21 @@ metadata:
 
 	framework.KubeDescribe("Kubectl logs", func() {
 		var nsFlag string
-		var rc string
-		containerName := "redis-master"
+		podName := "logs-generator"
 		ginkgo.BeforeEach(func() {
-			ginkgo.By("creating an rc")
-			rc = commonutils.SubstituteImageName(string(readTestFileOrDie(redisControllerFilename)))
+			ginkgo.By("creating an pod")
 			nsFlag = fmt.Sprintf("--namespace=%v", ns)
-			framework.RunKubectlOrDieInput(rc, "create", "-f", "-", nsFlag)
+			framework.RunKubectlOrDie("run", podName, "--generator=run-pod/v1", "--image="+agnhostImage, nsFlag, "--", "logs-generator", "--log-lines-total", "100", "--run-duration", "20s")
```

Comment:
Could you add a comment describing what the image will log, the frequency, and the duration?